src/labels/CustomLabels.labels

```diff
@@ -2694,6 +2694,20 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Button text for copying existing Batch setup to a new Batch.</shortDescription>
         <value>COPY</value>
     </labels>
+    <labels>
+        <fullName>bgeDryRunButton</fullName>
```

Comment:
This probably shouldn't have "button" in it, so it can be semantically reused elsewhere without confusion.